spec/scenic/adapters/postgres_spec.rb

```diff
@@ -51,16 +59,42 @@ module Adapters
 
           expect(adapter.views.map(&:name)).not_to include("greetings")
         end
+
+        it "raises an exception if the version of PostgreSQL is too old" do
+          adapter = Postgres.new
+          allow(ActiveRecord::Base.connection).to receive(:select_value) { "PostgreSQL 9.2.2 on x86_64-unknown-linux-gnu, compiled by gcc (Ubuntu/Linaro 4.6.3-1ubuntu5) 4.6.3, 64-bit" }
+
+          expect { adapter.drop_materalized_view("greetings") }
+            .to raise_error StandardError
+        end
       end
 
       describe "#refresh_materialized_view" do
-        it "raises descriptive error if concurrent refresh is not possible" do
+        it "raises an exception if the version of PostgreSQL is too old" do
           adapter = Postgres.new
-          adapter.create_materialized_view(:tests, "SELECT text 'hi' as text")
+          allow(ActiveRecord::Base.connection).to receive(:select_value) { "PostgreSQL 9.2.2 on x86_64-unknown-linux-gnu, compiled by gcc (Ubuntu/Linaro 4.6.3-1ubuntu5) 4.6.3, 64-bit" }
+
+          expect { adapter.refresh_materialized_view(:tests) }
+            .to raise_error StandardError
+        end
+
+        context "refreshing concurrently" do
+          it "raises descriptive error if concurrent refresh is not possible" do
+            adapter = Postgres.new
+            adapter.create_materialized_view(:tests, "SELECT text 'hi' as text")
+
+            expect { adapter.refresh_materialized_view(:tests, concurrently: true) }
+              .to raise_error(/Create a unique index with no WHERE clause/)
+          end
+
+          it "raises an exception if the version of PostgreSQL is too old" do
+            adapter = Postgres.new
+            allow(ActiveRecord::Base.connection).to receive(:select_value) { "PostgreSQL 9.3.2 on x86_64-unknown-linux-gnu, compiled by gcc (Ubuntu/Linaro 4.6.3-1ubuntu5) 4.6.3, 64-bit" }
```

Comment:
Nice. You can appease hound here by using `.and_return("...")` on a new line, indented once below `allow`.